src/debug.c

```diff
@@ -53,6 +55,15 @@ typedef ucontext_t sigcontext_t;
 #endif
 #endif
 
+/* Globals */
+int bug_report_start = 0; /* True if bug report header was already logged. */
```

Comment:
@oranagra consider making this static (maybe a good opportunity to make other func defs in this file static as well).